app/views/movies/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Movies Show Page</h1>
+
+<h2><%= @movie.title %></h2>
+<p>Year Released: <%= @movie.creation_year %></p>
+<p>Genre: <%= @movie.genre %></p>
+
+
+ 
+<% @movie.actor_sort.each do |actor| %>
+  <li><%= actor.name %>
+  <%= actor.age %></li>
+<% end %>
+
+<p>Average Age of Actors: <%= @movie.actor_age_avg %>
+
+<p> Add an Actor to This Movie:</p>
+<%= form_with url: "/movies/#{@movie.id}", method: :patch, local: true, data: {turbo: false} do |f| %>
```

Comment:
As discussed, a more RESTful option could be a POST to "/movies/:id/actors" and have that route to a joins controller